src/Documentation/doc/advanced/corporation/faq.md

```diff
@@ -204,6 +209,12 @@ You have to enable it.
 
 Yes, you can reimplement Market-TA2. Implementing a custom Market-TA2 script is the best optimization in round 3+. Check this [section](./optimal-selling-price-market-ta2.md) to see how to do it.
 
+### Why can I not sell all produced goods in the storage even after Market-TA2 sets the optimal price?
+
+In the SALE state, the game calculates `MaxSalesVolume` of the material/product. This is the number of items that can be sold in this state. This value is the product of many multipliers, and one of those multipliers is the penalty modifier mentioned above. Market-TA1 and Market-TA2 only guarantee that this multiplier does not affect you negatively (keep the penalty modifier being 1). They do not increase `MaxSalesVolume` and help you sell more items.
```

Comment:
I would maybe reword this a little bit. You *can* sell more items by reducing the price. The issue is the shape of the MarkupModifier function and the way TA2 works.

MarkupModifier is a piecewise function that is 1e12 at 0, `marketPrice/sellingPrice` from 0 < sellingPrice <= marketPrice, 1 from marketPrice < sellingPrice <= marketPrice + marketLimit, and `(markupLimit / (sellingPrice - marketPrice))^2` past that.

Since TA2 always assumes you are in the last section, when it tries to invert the function it maps anything lower into the constant "1" section. It would be entirely possible to create a proper inverse so that it *would* boost sales when needed, it simply does not.